sdks/go/pkg/beam/io/fhirio/common.go

```diff
@@ -32,8 +38,54 @@ const (
 	baseMetricPrefix = "fhirio/"
 )
 
+func executeRequestAndRecordLatency(ctx context.Context, latencyMs *beam.Distribution, requestSupplier func() (*http.Response, error)) (*http.Response, error) {
```

Comment:
@youngoli, follow up to [this comment](https://github.com/lnogueir/beam/pull/24#discussion_r896060946), I got rid of the generic return value here because I double checked that the response would always be a `*http.Response`.

Reply:
Good thinking 👍